proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -0,0 +1,103 @@
+# Project Name
+
+## Project Champion
+
+Who is the main point of contact during the application process?
+
+## CPC Champion (optional)
+
+If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
```

Comment:
Any thoughts about this person being selected from voting members, or also non-voting members?

Reply:
My idea was it would be whomever volunteered

Reply:
Comment resolved during #168